### PR TITLE
 Remove PINGS_DIR const from kotlin code

### PR DESCRIPTION
**Pull request by brizental**
Fixes [Bug 1620672](https://bugzilla.mozilla.org/show_bug.cgi?id=1620672)

### CODE REVIEW

File: glean-core/android/src/test/java/mozilla/telemetry/glean/GleanTest.kt [hunk 1]
```diff
@@ -385,27 +380,6 @@ class GleanTest {
         }
     }
 
-    @Test
```
Comment (brizental) [inline glean-core/android/src/test/java/mozilla/telemetry/glean/GleanTest.kt:388]: This was testing the same thing as the `send a ping`.

File: glean-core/android/src/test/java/mozilla/telemetry/glean/GleanTest.kt [hunk 2]
```diff
@@ -104,17 +101,15 @@ class GleanTest {
...
-        // Now trigger it to upload
+        // Trigger it to upload
         triggerWorkManager(context)
 
         val request = server.takeRequest(20L, TimeUnit.SECONDS)
         val docType = request.path.split("/")[3]
         assertEquals("baseline", docType)
+
+        // Make sure baseline was the only ping on queue
+        assertEquals(server.takeRequest(20L, TimeUnit.SECONDS), null)
```
[the remark this answers is not in the document]
Comment (badboy) [reply glean-core/android/src/test/java/mozilla/telemetry/glean/GleanTest.kt:112]: Yeah, we should probably in general have a lower default than 20s in tests.

cc @Dexterp37

[the remark this answers is not in the document]
Comment (brizental) [reply glean-core/android/src/test/java/mozilla/telemetry/glean/GleanTest.kt:112]: This timeout is also optional. We can just remove it.

[the remark this answers is not in the document]
Comment (Dexterp37) [reply glean-core/android/src/test/java/mozilla/telemetry/glean/GleanTest.kt]: +1 on not using `takeRequest` at all!

**Pull request merged**